packages/webapp/src/containers/Navigation/SlideMenu.jsx

```diff
@@ -15,7 +15,7 @@
 
 import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
-import PureSlideMenu from '../../components/Navigation/slideMenu';
+import PureSlideMenu from '../../components/Navigation/SlideMenu';
```

Comment:
For some reason the folder name is actually lowercase for the component

Reply:
Changed the name to side menu, hopefully that'll do the trick